corona-warn-app/cwa-app-android
# Download ccl config and booster rules again if they failed earlier today (DEV)

This PR fixes the issue that the `lastExecutionTime` of `CCLConfigurationUpdater` was set although there were failing network requests (either the one to ccl-config or booster rules or both). 

This meant that there were no new retries to get new data for the current day when the app went to foreground. 

This is fixed now. `lastExecutionTime` is now only set if both network requests were successful.

How to test:
- fresh install
- activate airplane mode
- start app
- see failing network requests 
- deactivate airplane mode
- bring app back to foreground
- network requests should be performed again

More Information: https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/logic-engine/docs/spec/dcc-ccl.gen.md#api-for-updating-the-ccl-configuration-from-cdn
